x-pack/plugins/security_solution/public/management/pages/event_filter/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListType } from '../../../../../lists/common/shared_exports';
+
+export const EVENT_FILTER_LIST_ID = '.endpointEventFilterList';
+export const EVENT_FILTER_LIST_TYPE: ExceptionListType = 'endpoint_events';
```

Comment:
Should get this value from the List plugin (I think they have a common export there for it). the value should be `endpoint`

Reply:
Maybe I'm wrong then, but I thought we had created a new ExceptionListType for that in [this](https://github.com/elastic/kibana/pull/96037) PR

Reply:
Yeah, I remember maybe commenting that as far as data storage, we use the `endpoint` type currently. I was not sure if you added that new type in order to support the UI. If not being used, then lets plan to remove it.